crates/bevy_core/src/serde.rs

```diff
@@ -39,3 +40,32 @@ impl<'de> Visitor<'de> for EntityVisitor {
         Ok(Name::new(v))
     }
 }
+
+impl Serialize for FrameCount {
+    fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        serializer.serialize_str(self.0.to_string().as_str())
+    }
+}
+
+impl<'de> Deserialize<'de> for FrameCount {
+    fn deserialize<D: Deserializer<'de>>(deserializer: D) -> Result<Self, D::Error> {
+        deserializer.deserialize_u32(FrameVisitor)
+    }
+}
+
+struct FrameVisitor;
```

Comment:
Since this is a manual implementation of Serde serialisation, I think it would be nice if you could include a unit test just to verify it works as intended.

Reply:
sure. I included a test for `Name` as well while I was at it. n.b. I have added `serde_test` as a dev dependency for this.